unitTests/rest/SearchEndpointTest.ts

```diff
@@ -310,6 +311,48 @@ export function SearchEndpointTest() {
           });
         });
 
+        it('for plan', done => {
```

Comment:
The other tests in this file are similar to this one, but on the other hand, they're kind of "old"/not extremely well written and could be improved. 

So I'd vote to at least make these new tests easier to read.

* I think we could split up the test with one expectation each, under the same `describe` block that does the test setup.
* Could remove .then() for async instead (the `promiseSuccess` variable).

If you think it's better to keep all the tests "similar" in the same file, then I can also understand that, and I won't fight ;)

Reply:
I will split them a bit but I won't go overboard with many changes 😛